std_navbar-prerelease.ddoc

```diff
@@ -121,6 +121,17 @@ $(DIVID cssmenu, $(UL
 		  $(MODULE3 core, sync, semaphore)
 		)
 	  )
+
+	$(MENU_W_SUBMENU $(TT experimental))
+	  $(ITEMIZE
+		$(PACKAGE $(PACKAGE_NAME logger),
+		  $(MODULE4 std, experimental, logger, package),
+		  $(MODULE4 std, experimental, logger, core),
+		  $(MODULE4 std, experimental, logger, filelogger),
+		  $(MODULE4 std, experimental, logger, multilogger),
+		  $(MODULE4 std, experimental, logger, nulllogger)
+		)
+	  )
```

Comment:
OK, though that's a little inconsistent (package name is `std.experimental`, not `experimental`).